### ISSUE TITLE
Foreign Schools are displaying Housing Allowance when they are not supposed to

### ISSUE THREAD

**Issue by mrblanco1** [department-of-veterans-affairs/va.gov-team#38544]
## Issue Description
Foreign schools should NOT display housing allowance when:

Use case:

Actual:
University of Toronto. "I chose Will you be taking any classes in person? "No"". I also choose "Did you use your Post-9/11 GI Bill...."No"". Under Estimated benefits per term / Housing allowance: Fall $4052 - Sprint $4052. 

Expected:
U of Toronto, we wouldn't pay for any training that was distance only. (not just housing- no payments at all).

Rule:
Foreign schools are based on the facility code. It's not all schools physically located outside of the US. There are US branches of foreign schools outside of the US. but those can receive payment for distance learning. 

HINT: It is believed that the solution will be with the facility code...the last 2 digits tell us the location.

Example: my 1st example is an extension site in Japan (pretty sure extensions don't show up at all. TEMPLE UNIVERSITY - JAPAN CAMPUS - 11X16038

This "extension" can get paid for distance and should display housing allowance in CT. 

-----------------------------------------------------
Two more examples: 
1. Branch from NY: FASHION INSTITUTE OF TECHNOLOGY-KOREA - 14000232
2. Branch from a NY school: NEW YORK UNIVERSITY - ABU DHABI - 318N0132

These two "extensions" can get paid for distance and should display housing allowance in CT. 

-------------------------------------------------------

We are almost sure the solution will be with the facility code...the last 2 digits tell us the location

---------------------------------------------------------
This is incorrect as the University of Toronto is a Foreign Institution and should not display/calculate housing allowance.

![image.png](https://images.zenhubusercontent.com/60e78c6b02152f0a04637267/bfed4f84-1c78-41af-a1cb-2f2133609e64)

**Comment by dsrr47** [comment]
There is an additional issue where the value of the radio buttons for "Will you be taking classes in person had values opposite to their label. Ex the Yes radio button has a "value=no" property. Despite being miss labeled the functionality works correctly.

**Comment by dsrr47** [comment]
> Expected:
> U of Toronto, we wouldn't pay for any training that was distance only. (not just housing- no payments at all).
Does this also mean $0 for book stipends?

**Comment by mrblanco1** [comment]
> > Expected:
> > U of Toronto, we wouldn't pay for any training that was distance only. (not just housing-  no payments at all).
> 
> @mrblanco1 Does this also mean $0 for book stipends?

Hi @dsrr47 - I asked Cinda and her response was "T&F's and the book stipends is payable". (hope this helps)

**Comment by dsrr47** [comment]
@mrblanco1  Okay, in the example given, University of Toronto, the tuition and fee's is saying $0 payable is that correct?

**Comment by mrblanco1** [comment]
> @mrblanco1 Okay, in the example given, University of Toronto, the tuition and fee's is saying $0 payable is that correct?
> 
> And to be clear payable means they do receive the book stipend and tuition and fees? That seems to contradict the earlier "no payments at all" statement.

That is correct @dsrr47 - Cinda provided more details and stated "payable means they do receive the book stipend and tuition and fees". I will update the description.

**Comment by dsrr47** [comment]
@mrblanco1 Okay, one more thing. If they _are_ attending the school in person would they then get a housing allowance?

**Comment by dsrr47** [comment]
The issue I mentioned [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/38544#issuecomment-1069362976) about the radio button values being opposite Is fixed.

**Comment by mrblanco1** [comment]
> @mrblanco1 Okay, one more thing. If they are attending the school in person would they then get a housing allowance? And how will I know if the amount being calculated is correct? Using the University of Toronto as an example can I get a confirmation on what the actual allowance should be?

Hi David, let me find that for you.

**Comment by mrblanco1** [comment]
> > @mrblanco1 Okay, one more thing. If they are attending the school in person would they then get a housing allowance? And how will I know if the amount being calculated is correct? Using the University of Toronto as an example can I get a confirmation on what the actual allowance should be?
> 
> Hi David, let me find that for you.

HI David!

This is the latest feedback from Cinda:

yes, in person training in a degree program is the only type of training that is approvable at a foreign school. 
like 1

all foreign degree training programs receive the national average rate. currently that rate is 1,801.00 for someone eligible at the 100% tier. rates change each 8/1 and are good thru the following 7/31
like 1

US schools located overseas also get the national rate for in person training, but can also get the distance rate (half the national rate)

**Comment by dsrr47** [comment]
> There is an additional issue where the value of the radio buttons for "Will you be taking classes in person had values opposite to their label. Ex the Yes radio button has a "value=no" property. Despite being miss labeled the functionality works correctly.

@mrblanco1  I want to point out my first comment on this thread and say that this is still an accessibility issue. I tried simply swapping the values but that caused quite a few linter checks to fail and should probably be made an issue on its own.

**Comment by jfinnern** [comment]
Validated in staging
Test run: https://dsvavsp.testrail.io/index.php?/runs/view/3129&group_by=cases:section_id&group_order=asc
